joomla/joomla-cms
# com_tags // avoid deprecation message is $element->core_body is empty

Deprecation messages are shown for some tagged elements: 
![image](https://user-images.githubusercontent.com/3834825/212553903-d29d036f-ff70-429c-b25c-89a189a5365f.png)



Pull Request for Issue # .

### Summary of Changes
In com_tags, a tagged element's text-variable is transformed into null if the value is empty. 
That causes deprecation messages in plugins like loadmodules and emailcloak. They usually don't need to check for null values in $article->text. 

### Testing Instructions

1. configure PHP to show deprecation messages
2. create a new contact using com_contact: just enter a name and tag it with "testtag"
3. open the frontend /index.php?option=com_tags&view=tags
4. click on the new tag 'testtag'.
5. see the deprecation messages.


### Actual result BEFORE applying this Pull Request

Following the test instructions, deprecation messages are shown


### Expected result AFTER applying this Pull Request

Following the test instructions, deprecation messages are now shown


### Link to documentations
Please select:
